fakes/get_etc_passwd_file_content.go

```diff
@@ -0,0 +1,7 @@
+package fakes
```

Comment:
@pacostas I don't think this the way that fakes are generally handled. They are typically automatically generated and I don't see the statements in generate.go that would do that. Maybe I just missed it though?

Reply:
It's a line like this that I'd be looking for 

//go:generate faux --interface DependencyManager --output fakes/dependency_manager.go but for a different interface

Reply:
Maybe there is more than one way they are used/generated. Did you see an instance were there was a fake in one of the other buildpacks were there was no generate faux line ?

Reply:
I see, yes you are right, this doesnt quite follow the pattern of fakes. Looking at it.

Reply:
I removed it from fakes, and mocking it instead before passing it as an argument on the tests. https://github.com/paketo-community/ubi-nodejs-extension/pull/26/commits/e182221a4e422fa964a79199ec0a15b6da2257ed

The tests are having a hard time to pass, as it fails to download the go packages (before running the tests). This is irrelevant with the pr. The only way to re-run them is to push new changes by just adding/removing the dot character on the commit message.